tests/unit/components/WalletBase.spec.ts

```diff
@@ -3,7 +3,7 @@ import Theme from '@soramitsu/soramitsu-js-ui/lib/types/Theme';
 
 import { useDescribe, useMount } from '../../utils';
 import WalletBase from '@/components/WalletBase.vue';
-
+// TODO: [vuex]
```

Comment:
Is this TODO for the future or current PR?

Reply:
removed, sorry